stories/usecases/introduction.mdx

```diff
@@ -1,11 +1,7 @@
 import { Meta } from '@storybook/addon-docs';
+import LinkTo from '@storybook/addon-links/react';
```

Comment:
Is this import part of the migration script?
the import is unused in this file.

Reply:
No. I think I have just placed it there manually because I tried something. It seems, that I forgot to delete it again :)